accepted/2023/simplify-output-paths.md

```diff
@@ -44,12 +44,12 @@ To override the default artifacts folder path, the `ArtifactsPath` property can
 
 Under the artifacts folder will be nested folders for the type of the output, the project name, and the pivots, ie `<ArtifactsPath>\<Type of Output>\<Project Name>\<Pivots>`.
 
-- Type of Output - This will be a value such as `bin`, `publish`, `intermediates`, or `package`.  Projects will be able to override this value, for example to separate shipping and non-shipping artifacts into different folders
+- Type of Output - This will be a value such as `bin`, `publish`, `obj`, or `package`.  Projects will be able to override this value, for example to separate shipping and non-shipping artifacts into different folders
   - `bin` will be the folder for the normal build output.  Projects can override this with `ArtifactsOutputName`
   - `publish` will be the folder for the publish output.  Projects can override this with `ArtifactsPublishOutputName`
   - `package` will be the folder for the package output, where .nupkg folders are placed. (Or should it be `packages`?  Feedback welcome.)  Projects can override this with `ArtifactsPackageOutputName`
```

Comment:
Do you mean `.nupkg packages` or is `.nupkg folders` correct? Is the intent that each project would get it's own folder in this folder or would the `.nupkg` files be in the root of the `package` folder?

Also, it should probably be clarified that this is **not** the directory where nuget packages get downloaded when doing a package restore, especially since there used to be a folder called `packages` which contained the cache of restored nuget packages (when using `packages.config`).